modules/distribution/src/repository/resources/dbscripts/apimgt/oracle.sql

```diff
@@ -493,14 +495,14 @@ CREATE OR REPLACE TRIGGER IDP_ROLE_TRIG
                END;
 /
 CREATE TABLE IDP_ROLE_MAPPING (
-            ID INTEGER,
-            IDP_ROLE_ID INTEGER,
-            TENANT_ID INTEGER,
-            USER_STORE_ID VARCHAR (253),
-            LOCAL_ROLE VARCHAR(253),
-            PRIMARY KEY (ID),
-            UNIQUE (IDP_ROLE_ID, TENANT_ID, USER_STORE_ID, LOCAL_ROLE),
-            FOREIGN KEY (IDP_ROLE_ID) REFERENCES IDP_ROLE(ID) ON DELETE CASCADE)
+			ID INTEGER,
+			IDP_ROLE_ID INTEGER,
+			TENANT_ID INTEGER,
+			USER_STORE_ID VARCHAR (253),
+			LOCAL_ROLE VARCHAR(253),
+			PRIMARY KEY (ID),
+			UNIQUE (IDP_ROLE_ID, TENANT_ID, USER_STORE_ID, LOCAL_ROLE),
+			FOREIGN KEY (IDP_ROLE_ID) REFERENCES IDP_ROLE(ID) ON DELETE CASCADE)
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04